backend/addcorpus/migrations/0005_add_validators.py

```diff
@@ -4,6 +4,11 @@
 from django.db import migrations, models
 
 
+def validate_image_filename_extension(filename):
+    allowed = ['.jpeg', '.jpg', '.png', '.JPG']
+    addcorpus.validation.creation.validate_filename_extension(filename, allowed)
```

Comment:
I would move this to `addcorpus.validation` somewhere, hard to find in migration files.

Reply:
Also, using MIMETYPE is a better way to validate file type, but this will almost certainly work fine.

Reply:
I moved it to the migration files because it's not used anymore; ImageField does this validation out-of-the-box. It's still referenced in this old migration, though.